### PR TITLE
doc/man/cephfs-top.rst: add missing options: --delay, --conffile

### PR DESCRIPTION
**Pull request by rishabh-d-dave**
add missing options: --delay, --conffile

Signed-off-by: wangxinyu <wangxinyu@inspur.com>
Signed-off-by: Rishabh Dave <ridave@redhat.com>

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

**Comment by rishabh-d-dave** [comment]
@kotreshhr @joscollin @nmshelke @lxbsz @anthonyeleven Please re-review and/or approve this PR. It is same as #45572 that was reviewed/approved by you earlier.

File: doc/man/8/cephfs-top.rst [hunk 1]
```diff
@@ -36,6 +36,14 @@ Options
 
    Perform a selftest. This mode performs a sanity check of ``stats`` module.
 
+.. option:: --conffile [CONFFILE]
+
+   Path to cluster configuration file
+
+.. option:: -d [DELAY], --delay [DELAY]
+
+   Metric refresh interval in seconds (default: 1 second)
```
Comment (joscollin) [inline doc/man/8/cephfs-top.rst:45]: `Metrics refresh interval` instead?

Comment (anthonyeleven) [reply doc/man/8/cephfs-top.rst:45]: Actually I think it would be better as just "Refresh interval in seconds (default: 1)"

**Review by anthonyeleven — APPROVED** [review]

**Comment by dparmar18** [comment]
LGTM apart from the suggested change.

**Review by joscollin — APPROVED** [review]

**Comment by joscollin** [comment]
jenkins test windows

**Comment by rishabh-d-dave** [comment]
@wxypro Thanks for the contribution!

**Pull request merged**